webapp/lib/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "pyroscope",
+  "version": "0.0.1-%VERSION%",
+  "main": "index.lib.js",
```

Comment:
I don't think this is right, I only see lib.js and such in the package (no index.lib.js).